python/tvm/relay/frontend/tflite.py

```diff
@@ -659,7 +659,23 @@ def _convert_reduce(self, relay_op, op):
         reduce_options.Init(op_options.Bytes, op_options.Pos)
         keep_dims = reduce_options.KeepDims()
 
+        if input_tensor.qnn_params:
+            in_expr = _qnn.op.dequantize(data=in_expr,
```

Comment:
Hmm...do we have to dequantize -> compute fp32 -> requantize? could we do only integer computation?

Reply:
Similar concern as that of @FrozenGene 
From maths size, it looks safe to me, but let me know if it wrong

~~~~
Initial Quantized tensor - QA - shape (N, dtype='int8') 

Mean would be 

scale_output * (Qout - zp_out) = scale_in * [(QA[0] + QA[1] .... + QA[N-1]) - N* zp_in]/N

scale_output * (Qout - zp_out) = scale_in * (Mean(QA) - zp_in)
~~~~

So, basically, upcast the quantized tensor to int32, call Int Mean, and then requantize if the output scale/zp are different.

Reply:
Updated as suggested above